components/gitpod-cli/cmd/prebuild-logs.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/gitpod-io/gitpod/gitpod-cli/pkg/theialib"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// initCmd represents the init command
+var prebuildLogs = &cobra.Command{
+	Use:   "prebuild-logs",
+	Short: "Opens logs of Gitpod prebuilds",
+	Long: `
+Opens all logs of Gitpod prebuilds.
+Number of logs depends upon the init tasks you have configured.
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		const prebuildFilePath = "/workspace/.gitpod/prebuild-log-*"
```

Comment:
it is implementation detail of supervisor, i think it should be a secret of supervisor how to store and fetch content. Clients should not rely on it.

See https://en.wikipedia.org/wiki/Information_hiding